docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -150,3 +150,58 @@ executors. For example, if `host1` contains `blockA` and a job using `blockA` is
 cluster with `--num-executors=1`, Spark might place the only executor on `host2` and have poor locality.
 However, if `--num-executors=2` and executors are started on `host1` and `host2`, Spark will be smart
 enough to prioritize placing the job on `host1`.
+
+## `Failed to login` Issues with Spark Shell
+
+To run the `spark-shell` with the Alluxio client, the Alluxio client jar will have to be added to the classpath of the
+Spark driver and Spark executors, as described earlier. However, sometimes Alluxio will fail to determine the security
+user and will result in an error message similar to: `Failed to login: No Alluxio User is found.`
+
+### Configure `spark.sql.hive.metastore.sharedPrefixes` for Spark
```

Comment:
not clear to me if three options are all required or picking one option is good.
Can we be more specific here?